src/main/java/orca/flukes/ndl/ManifestLoader.java

```diff
@@ -227,12 +227,30 @@ private String getPrettyName(Resource r) {
 		return rname;
 	}
 	
-	private void addNodeToInterface(String iface, OrcaNode n) {
-		List<OrcaNode> others = interfaceToNode.get(iface);
+	Set<Resource> sameAs = new HashSet<>();
+	
+	private void addNodeToInterface(Resource iface, OrcaNode n) {
+
+		String ifaceName = getTrueName(iface);
+		GUI.logger().debug("Considering adding interface " + ifaceName + " to the list");
+		
+		Resource sameInt = NdlCommons.getSameAsResource(iface);
```

Comment:
`sameInt` could probably be better named, to match existing conventions in this code, and so as not to be confused with an Integer.  perhaps `sameIface` ?